src/api/app/models/package.rb

```diff
@@ -929,12 +929,10 @@ def buildresult(prj = project, show_all = false)
   end
 
   def jobhistory_list(project, repository, arch, options = {})
-    options[:limit] = 100 if options[:limit].blank?
     options[:package] = name if options[:package].blank?
 
     results = Jobhistory.find_hashed(project: project.name, package: options[:package],
-                                     repository: repository, arch: arch,
-                                     limit: options[:limit])
```

Comment:
Did you verify that the default limit is 100?

Reply:
what do you mean?

Reply:
Before your change the frontend would always set it to 100. Now that this is gone what default limit would the backend use?

Reply:
now I get it, I am removing the limit :see_no_evil: